docs/content/en/tasks/manage/private-registries/_index.md

```diff
@@ -0,0 +1,393 @@
+
+## Image coordinates
+
+K8ssandra 1.3.0 adds the ability to fully specify coordinates for every image that it deploys. To better understand what we mean by image coordinates, let's look closer.
+
+### Format of image coordinates
+
+Consider this example:
+
+```
+docker.io/k8ssandra/cass-management-api:3.11.10-v0.1.26
+```
+
+* `docker.io` is the registry.
+
+* `k8ssandra` is the repository.
+
+* `cass-management-api` is the image.
+
+* `3.11.10-v0.1.26` is the image tag.
+
+Given this, the complete **format** of coordinates looks like this:
+
+```
+registry/repository/image:tag
+```
+
+K8ssandra allows you to configure each of these parts for each image that it deploys.
+
+### Chart properties YAML
+
+For each container that K8ssandra deploys, there are chart properties like this:
+
+```yaml
+<container-name>:
+  image:
+    registry:
+    repository:
+    tag:
+    pullPolicy: 
+```
+
+Example YAML:
+
+```yaml
+cassandra:
+  serviceAccount: cassandra
+  image:
+    registry: myregistry
+    repository: myrepo/cass-management-api
+    tag: 3.11.10-v0.1.26
+    pullPolicy: IfNotPresent
+```
+
+{{% alert title="Tip" color="success" %}}
+While `pullPolicy` is shown above, it is not part of the image coordinates. Possible values for `pullPolicy` are:
+
+* `IfNotPresent` (default)
+* `Always` (default when the image tag is `latest`)
+* `Never`
+
+See [ImagePullPolicy](https://helm.sh/docs/chart_best_practices/pods/#imagepullpolicy) in the Helm v3 documentation.
+{{% /alert %}}
+
+### List of chart properties
+
+Here is a list of chart properties that correspond to K8ssandra deployed containers having an `image` property:
+
+| Chart property / container name                          | Summary                               |
+| -------------------------------------------------------- | ------------------------------------- |
+| `cassandra`                                              | Runs the Management API and Cassandra&reg; |
+| `cassandra.configBuilder`                                | An init container that provisions Cassandra configuration files such as cassandra.yaml |
+| `cassandra.jmxCredentialsConfig` | An init container that configures authentication for remote JMX access |
+| `cassandra.loggingSidecar` | A container that tails Cassandra's system.log file |
+| `stargate` | Runs Stargate, which is an API for interacting with Cassandra |
+| `stargate.waitForCassandra` | An init container that waits until all the Cassandra pods are ready |
+| `reaper` | Run Reaper, which provides repair functionality for Cassandra data |
+| `medusa` | Runs the medusa container that performs backups, and the `medusa-restore` init container that performs restores |
+| `cleaner` | A Helm pre-delete hook that deletes the CassandraDatacenter |
+| `client` | Used in the `crd-upgrader pre-upgrade` to apply Custom Resource Definition (CRD) updates |
```

Comment:
I would change `crd-upgrader pre-upgrade` to `crd-upgrader` pre-upgrade hook

Reply:
Updated in subsequent commit - thanks John.